### PR TITLE
August updates

### PR DESCRIPTION
**Pull request by smare**
Updated changelog and API reference documentation.

### CODE REVIEW

File: report-results/docs/index.md [hunk 1]
```diff
@@ -525,11 +521,44 @@ The API returns an `HTTP 200 OK` response if the report is unavailable. The `err
 
 | Possible Cause | Solution |
```
Comment (smare) [inline report-results/docs/index.md:522]: Maybe use [this tool](https://www.tablesgenerator.com/markdown_tables#:~:text=Markdown%20tables%20support,syntax%20for%20creating%20simple%20tables.) to make the table look better?

Comment (mattgrogan) [reply report-results/docs/index.md:522]: sure go ahead

Comment (smare) [reply report-results/docs/index.md:522]: Okay, go ahead and merge it and I'll update the table after

**Comment by mattgrogan** [comment]
I don’t want to merge until the changes are live.

**Comment by smare** [comment]
Prod is deployed

**Pull request merged**